sdks/java/maven-archetypes/pom.xml

```diff
@@ -87,6 +87,29 @@
           </execution>
         </executions>
       </plugin>
+      <plugin>
+        <artifactId>maven-clean-plugin</artifactId>
```

Comment:
It would make more sense for this configuration to live in the module that generates it. Imagine that someone does "mvn clean" in the underlying module -- it wouldn't clean it.